generated/nidaqmx/_task_modules/timing.py

```diff
@@ -2892,6 +2892,7 @@ def sync_pulse_type(self):
             self._handle)
         check_for_error(error_code)
 
+
     def cfg_burst_handshaking_timing_export_clock(
```

Comment:
https://github.com/ni/nidaqmx-python/blob/master/src/codegen/templates/property_deprecated_template.py.mako adds an extra newline. I think the fix is to add a backslash to the last line: `</%def>\`

Reply:
I have fixed this issue by adding a `\` in a `def` statement in the `property_deprecated_template.py.mako`, which seemed to be causing the issue.

Reply:
This fix also removes unwanted lines that were generated in `ai_channel.py` and `ci_channel.py`.